lib/core.ps1

```diff
@@ -54,27 +54,33 @@ function get_config($name, $default) {
     return $scoopConfig.$name
 }
 
-function set_config($name, $value) {
-    if($null -eq $scoopConfig -or $scoopConfig.Count -eq 0) {
-        ensure (Split-Path -Path $configFile) | Out-Null
-        $scoopConfig = New-Object PSObject
+function set_config {
+    Param (
+        [ValidateNotNullOrEmpty()]
+        $name,
+        $value
+    )
+
+    if ($null -eq $scoopConfig -or $scoopConfig.Count -eq 0) {
+        $null = ensure (Split-Path -Path $configFile)
```

Comment:
This function doesn't get called often. A millisecond performance boost would not make any difference.
I would go for readability in this case and use `| Out-Null`

Reply:
I personally find both equally readable and if we were to standardize the entire codebase on one method of discarding STDOUT choosing the most performant one could add up.

I can see how that should be a seperate issue/PR though, so for the time being I'll change it back.